packages/adapter-netlify/README.md

```diff
@@ -69,7 +83,15 @@ During compilation, redirect rules are automatically appended to your `_redirect
 
 ### Using Netlify Functions
 
-With this adapter, SvelteKit endpoints are hosted as [Netlify Functions](https://docs.netlify.com/functions/overview/). Netlify function handlers have additional context, including [Netlify Identity](https://docs.netlify.com/visitor-access/identity/) information. You can access this context via the `event.platform.context` field inside your hooks and endpoints.
+With this adapter, SvelteKit endpoints are hosted as [Netlify Functions](https://docs.netlify.com/functions/overview/). Netlify function handlers have additional context, including [Netlify Identity](https://docs.netlify.com/visitor-access/identity/) information. You can access this context via the `event.platform.context` field inside your hooks and `+page.server` or `+layout.server` endpoints. These are [serverless functions](https://docs.netlify.com/functions/overview/) when the `edge` property is `false` in the adapter config or [edge functions](https://docs.netlify.com/edge-functions/overview/#app) when it is `true`.
+
+```js
+// +page.server.js
+export const load = async (event) => {
+   const context = event.platform.context
+	 console.log(context)  // shows in your functions log in the Netlify app
```

Comment:
Not a native speak, but this sounds better to me. Feel free to ignore.
```suggestion
	 console.log(context)  // shows up in your functions log in the Netlify app
```

Reply:
Great catch, looks good to me.